### PR TITLE
add option to only play sound when window is focussed

### PR DESCRIPTION
**Pull request by kempanator**

### CODE REVIEW

**Comment by Zolhungaj** [comment]
It's not technically wrong. But could you change the spelling from "focussed" to "focused"?

**Review by Zolhungaj — CHANGES_REQUESTED** [review]
Focussed -> focused

**Comment by kempanator** [comment]
fixed

File: gameplay/amqShowPingPong.user.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // ==UserScript==
 // @name         AMQ Show Ping-Pong
-// @version      1.0.0
+// @version      1.0.1
```
Comment (Zolhungaj) [inline gameplay/amqShowPingPong.user.js:3]: Pretty sure this script stopped working when the game switched to socket-io 4

**Review by Zolhungaj — APPROVED** [review]
LGTM

**Pull request merged**